iac-extensions/arm/src/test/resources/checks/SubscriptionOwnerCapabilitiesCheck/Microsoft.Authorization_roleDefinitions/test.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "type": "Microsoft.Authorization/roleDefinitions",
+      "apiVersion": "2022-04-01",
+      "name": "Compliant - specific actions are specified",
+      "properties": {
+        "permissions": [
+          {
+            "actions": ["Microsoft.Compute/*"],
+            "notActions": []
+          }
+        ],
+        "assignableScopes": [
+          "[subscription().id]"
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.Authorization/roleDefinitions",
+      "apiVersion": "2022-04-01",
+      "name": "Compliant - scope is limited to a resource group",
+      "properties": {
+        "permissions": [
+          {
+            "actions": ["*"],
+            "notActions": []
+          }
+        ],
+        "assignableScopes": [
+          "/subscriptions/b24988ac-6180-42a0-ab88-20f7382dd24c/resourceGroups/b24988ac-6180-42a0-ab88-20f7382dd24c"
```

Comment:
```suggestion
          "${type}"
```
Same here.

Reply:
Here that's a compliant example with insensitive scope, but the test is parameterized with different types of sensitive scopes.